packages/dotcom-build-images/README.md

```diff
@@ -0,0 +1,56 @@
+# @financial-times/dotcom-build-images
+
+This package exports a Webpack plugin to provide support for static image assets in our asset pipeline.
+The plugin will copy all image files contained in the `client/` directory, and any of its subdirectories, into the configured public directory. 
+
+## Getting started
+
+This package is compatible with Node 12+ and is distributed on npm.
+
+```sh
+npm install --save-dev @financial-times/dotcom-build-images
+```
+
+After installing the module you must add it to the list of plugins in your project's `webpack.config.js` configuration file:
+
+```js
+const { PageKitImagesPlugin } = require('@financial-times/dotcom-build-base');
+
+module.exports = {
+	plugins: [
+		new PageKitImagesPlugin()
+	]
+}
+```
+
+## Arguments
+
+- `basePath` - Directory to look in for images. Defaults to `./client/`.
+
```

Comment:
the convention for this in other plugin docs is a table, e.g. https://github.com/Financial-Times/dotcom-page-kit/tree/master/packages/dotcom-build-js#options